<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QMainWindow" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QWidget" name="center">
   <layout class="QVBoxLayout" name="vlay">
    <property name="leftMargin">
     <number>48</number>
    </property>
    <property name="topMargin">
     <number>48</number>
    </property>
    <property name="rightMargin">
     <number>48</number>
    </property>
    <property name="bottomMargin">
     <number>48</number>
    </property>
    <item>
     <spacer name="vsp1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlay1">
      <item>
       <widget class="QLabel" name="label1">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="vsp2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QFormLayout" name="flay">
      <property name="horizontalSpacing">
       <number>36</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>帐号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ledit1">
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ledit2">
        <property name="maxLength">
         <number>16</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>类别</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout" name="hlay2">
        <item>
         <spacer name="hsp1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtn1">
          <property name="text">
           <string>学生</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hsp2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtn2">
          <property name="text">
           <string>商户</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hsp3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="vsp3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlay3">
      <item>
       <spacer name="hsp4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn1">
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hsp5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn2">
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hsp6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="vsp4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
